src/apps/survey/demoSurveyConfig.ts

```diff
@@ -87,11 +87,13 @@ export const config: ISurveyConfig = {
       },
     },
   },
-  // todo +nadgrids=@OSTN15_NTv2_OSGBtoETRS
   projections: [
-    [
-      'EPSG:27700',
-      '+proj=tmerc +lat_0=49 +lon_0=-2 +k=0.9996012717 +x_0=400000 +y_0=-100000 +ellps=airy +units=m +no_defs +type=crs',
-    ],
+    {
+      projection: [
+        'EPSG:27700',
+        '+proj=tmerc +lat_0=49 +lon_0=-2 +k=0.9996012717 +x_0=400000 +y_0=-100000 +ellps=airy +units=m +no_defs +nadgrids=OSTN15_NTv2_OSGBtoETRS',
+      ],
+      gridFileName: 'OSTN15_NTv2_OSGBtoETRS',
```

Comment:
What about giving an URL instead? It may be absolute: /src/helperFiles/OSTN15_NTv2_OSGBtoETRS.gsb,